units/ssh_keys_update_static.go

```diff
@@ -61,41 +56,4 @@ func (j *staticUpdateSSHKeysJob) Run(ctx context.Context) {
 		}
 		j.host = h
 	}
-	if j.host.Status != evergreen.HostRunning {
```

Comment:
Why was this job's logic removed? Without it, this job doesn't do anything.

Reply:
Same issue as I posted above. Fixed now